src/pages/variantEntity/TabFrequencies.tsx

```diff
@@ -122,35 +191,77 @@ const makeRowFromFrequencies = (frequencies: Frequencies) => {
 const makeInternalCohortsRows = (rows: StudyNode[]) =>
   rows.map((row: StudyNode, index: number) => ({ ...row.node, key: `${index}` }));
 
-const TabFrequencies = ({ variantId }: OwnProps) => {
-  const { loading, data, error } = useTabFrequenciesData(variantId);
+const mapDispatch = (dispatch: DispatchStoryPage) => ({
+  onClickStudyLink: (sqons: Sqon[]) => dispatch(createQueryInCohortBuilder(sqons)),
+});
+
+const mapState = (state: RootState): FrequencyTabTableContainerState => ({
+  currentVirtualStudy: state.currentVirtualStudy.sqons,
+});
+
+const connector = connect(mapState, mapDispatch);
 
-  if (error) {
+type PropsFromRedux = ConnectedProps<typeof connector>;
+
+type Props = OwnProps & PropsFromRedux;
+
+const TabFrequencies = (props: Props) => {
+  const { loading, data, error } = useTabFrequenciesData(props.variantId);
+
+  const studyIdList = data.studies.map((s: { node: { study_id: any } }) => s.node.study_id);
```

Comment:
check if a type can be added in models.ts. If it can be used elsewhere make a type before the component